pkgs/by-name/fa/factorio/package.nix

```diff
@@ -70,6 +70,15 @@ let
       nix-prefetch-url file://\''$HOME/Downloads/factorio_\''${releaseType}_x64_\''${version}.tar.xz --name factorio_\''${releaseType}_x64-\''${version}.tar.xz
 
     Note the ultimate "_" is replaced with "-" in the --name arg!
+
+    If you go this route you might want to tell nix to explicitly hold on to the
+    source tarball. Otherwise it could get GC'd from the nix store and you'd
```

Comment:
nix -> Nix

Reply:
did you miss the nix in the previous line, or are you drawing a deliberate distinction between the program `nix` in lowercase and the project `Nix` in uppercase or something?

Reply:
(btw if you use GitHub's "add a suggestion" button, it's slightly more convenient to credit you with these changes)

Reply:
Yes, I missed the first "nix". Yes, I think we should differentiate between the `nix` CLI and the Nix package manager. No, I don't think the first "nix" refers to the CLI.

(I don't need/want credit for such small changes -- I'd rather have the changes squashed -- but thanks for the reminder that it's a more precise way to suggest changes.)